website/pages/docs/developers/creating-new-plugin.md

```diff
@@ -9,4 +9,39 @@ More real world examples are:
 - [GCP Source Plugin](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/gcp)
 - [PostgreSQL Destination Plugin](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/postgresql)
 
-All other official source and destination plugins can be found [here](https://github.com/cloudquery/cloudquery/tree/main/plugins)
\ No newline at end of file
+All other official source and destination plugins can be found [here](https://github.com/cloudquery/cloudquery/tree/main/plugins)
```

Comment:
How about linking to https://www.cloudquery.io/docs/plugins/sources?

Our GitHub repo has some test plugins too

Reply:
I'd argue it's better to link to the code, since we are giving developers examples to reference. Even referencing the test plugins can be useful because of their simplicity (I know I have referenced them from time to time).

Reply:
Ah good point, so maybe remove the word "official" from "All other official".

Reply:
I rephrased it slightly 👍